src/index.tsx

```diff
@@ -1,10 +1,4 @@
 import ReactDOM from 'react-dom';
-import { HashRouter } from 'react-router-dom';
-import { App } from './App';
+import { Root } from './Root';
```

Comment:
Why do you add Routes here, that is the reason, your app is not working
You shouldn't have changed this file